addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -582,6 +582,9 @@ ListRenderer.include({
         // Freeze each th width according to their size in auto layout
         [...thead.getElementsByTagName('th')].forEach((th, index) => {
             const column = this.columns.find(col => col.attrs.name && col.attrs.name === th.dataset.name);
+            if (!column) {
```

Comment:
Beware of `<widget>` nodes, as they have a `name` attribute, but aren't fields. That being said, I don't know what to do for these nodes.